external/binplace/binplace.depproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <BinPlaceRef>true</BinPlaceRef>
+    <BinPlaceRuntime>true</BinPlaceRuntime>
+    <NETStandardVersion Condition="$(NuGetTargetMoniker.StartsWith('.NETStandard,Version=v'))">$(NuGetTargetMoniker.SubString(22))</NETStandardVersion>
+    <IsNetStandard Condition="'$(NETStandardVersion)' != ''">true</IsNetStandard>
```

Comment:
You don't need this, we already have [TargetsNETStandard](https://github.com/dotnet/corefx/blob/b3c2553cb2cf78257bd98e8eb7905ccc8dc1d7dd/Directory.Build.props#L169)